src/Command/Setup.php

```diff
@@ -55,15 +55,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $io = new SymfonyStyle($input, $output);
         $io->setDecorated(true);
-        $io->comment(sprintf('Composing your <info>%s</info> environment', (string) $envType));
+        $io->comment(sprintf('Start composing your <info>%s</info> environment', (string) $envType));
 
-        $vars = new AppVendor(
-            $io->ask('Vendor name', null, [AppVendor::class, 'validate']),
-            $io->ask('App name', null, [AppVendor::class, 'validate'])
-        );
+        $host = $io->ask('Host name', 'app', [Host::class, 'validate']);
```

Comment:
I'm afraid that people may input `[host].com` instead of `[host]` for instance, but I can't find the good wording for mentioning that `.dev` will be automatically appended to this value so there's no need for doing it yourself, while staying concise of course :) Any clue?